myuw/templates/handlebars/card/accounts/tuition.html

```diff
@@ -37,7 +37,7 @@
 
     {{#if has_balance}}
         {{#if is_credit}}
-            <div class="card-badge-container">
+            <div class="card-badge-container tuition">
```

Comment:
per our discussion about the .card-badge-container.tuiton syntax, examine whether you really need this class here. my sense is that you don't.